src/components/LibraryContainer.tsx

```diff
@@ -1,5 +1,4 @@
 /// <reference path="../../node_modules/@types/node/index.d.ts" />
-/// <reference path="../../node_modules/@types/whatwg-fetch/index.d.ts" />
```

Comment:
Is it possible to create a config file, that includes all the dependencies? You can inject the config file at the start up either using require or the react or webpack way.